apis/python/tests/test_soma_dataframe.py

```diff
@@ -18,11 +18,17 @@ def test_soma_dataframe_non_indexed(tmp_path):
     )
     sdf.create(schema=asch)
 
+    assert sorted(sdf.schema.names) == sorted(
+        ["foo", "bar", "baz", "soma_rowid", "soma_joinid"]
+    )
+    assert sorted(sdf.keys()) == sorted(sdf.schema.names)
+
     # ----------------------------------------------------------------
     # Write
     for _i in range(3):
         pydict = {}
         pydict["soma_rowid"] = [0, 1, 2, 3, 4]
+        pydict["soma_joinid"] = [0, 1, 2, 3, 4]
```

Comment:
can't hurt to make these `[100,400,500,200,300]` just to prove the point that they're truly arbitrary ...

Reply:
Good idea - fixed.